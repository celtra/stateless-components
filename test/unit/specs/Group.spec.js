import Group from '@/stateless/group.vue'
import Vue from 'vue'

const Constructor = Vue.extend(Group)
const vm = new Constructor({
    propsData: Group.usecases[0],
}).$mount()

describe('Group', () => {
    it('should match the snapshot', () => {
        expect(vm.$el).toMatchSnapshot()
    })
})
