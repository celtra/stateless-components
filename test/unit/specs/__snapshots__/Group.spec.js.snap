// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group should match the snapshot 1`] = `
<div
  class="group group--light"
>
  <div
    class="group__header"
  >
    <div
      class="group__label"
    >
      Modeling
    </div>
     
    <!---->
  </div>
   
</div>
`;
